src/mocks/syfomotebehov/mockSyfomotebehov.ts

```diff
@@ -2,23 +2,27 @@ import { motebehovMock } from "./motebehovMock";
 import { SYFOMOTEBEHOV_ROOT } from "@/apiConstants";
 import { VEILEDER_IDENT_DEFAULT } from "../common/mockConstants";
 import { http, HttpResponse } from "msw";
+import { MotebehovVeilederDTO } from "@/data/motebehov/types/motebehovTypes";
+
+let mockedMotebehov = motebehovMock;
 
 export const mockSyfomotebehov = [
   http.get(`${SYFOMOTEBEHOV_ROOT}/motebehov`, () => {
-    return HttpResponse.json(motebehovMock);
-  }),
-
-  http.post(`${SYFOMOTEBEHOV_ROOT}/motebehov/:fnr/behandle`, () => {
-    return new HttpResponse(null, { status: 200 });
+    return HttpResponse.json(mockedMotebehov);
   }),
 
-  http.post(`${SYFOMOTEBEHOV_ROOT}/motebehov/:fnr/behandle`, () => {
```

Comment:
Her har vi tydeligvis hatt to mocker for akkurat det samme endepunktet, men ingen av de hadde riktig path 🤷🏼‍♂️